python/ray/workflow/workflow_executor.py

```diff
@@ -227,59 +251,40 @@ def _iter_callstack(self, task_id: TaskID) -> Iterator[Tuple[TaskID, Task]]:
             yield task_id, state.tasks[task_id]
             task_id = state.task_context[task_id].creator_task_id
 
-    def _retry_failed_task(
```

Comment:
these functions are unused in the code (I forget to delete them before), so I just delete them